### DISCUSSION TITLE
Перестал работать.

### DISCUSSION THREAD

**Discussion by NikitaKulakov** [BlendLog/MinerSearch#2] [Q&A]
Видеокарта загружена на 100%, в диспетчере задач как будто ничего не происходит. Скачал Process Hacker, удаляется при запуске, да и сам диспетчер задач теперь удалён почему-то. Очевидно что майнер, но программа его не видит, увы.

**Comment by BlendLog** [comment]
напиши в лс в телегу @BlendLog
